### PR TITLE
Fixes #21617: Permissions are not preserved in shared-files on the relays

### PR DESCRIPTION
**Pull request by amousset**
https://issues.rudder.io/issues/21617

### CODE REVIEW

**Comment by amousset** [comment]
PR updated with a new commit

File: techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf [hunk 1]
```diff
@@ -115,6 +115,7 @@ bundle agent propagate_promises(trackingkey, technique_name, relay_sync_method,
       "${g.shared_files}"
         copy_from    => remote("${server_info.policy_server}","${g.shared_files}"),
         depth_search => recurse_visible("inf"),
+        perms => mog("640", "root", "rudder"),
```
Comment (amousset) [inline techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf:118]: not sure about this, we may need to preserver permissions instead

Comment (amousset) [reply techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf:118]: I'l check how it could work before

Comment (amousset) [reply techniques/system/rudder-service-relayd/1.0/relay/propagatePromises.cf:118]: Apparently we didn't preserve perms there back in 3.1, so let's assume we don't need it.

**Review by VinceMacBuche — APPROVED** [review]

**Pull request merged**